docs/faq.rst

```diff
@@ -709,6 +709,31 @@ Send a private email to the :ref:`security contact <security-contact>`
 if you think you have discovered a security issue.
 Please disclose security issues responsibly.
 
+How important is path/to/repo/nonce?
+------------------------------------
```

Comment:
do we maybe also want to mention the specific path where the local nonce is kept?

Reply:
done